README.md

```diff
@@ -313,6 +313,7 @@ A curated list of awesome resources, learning materials, tools, frameworks, plat
   * [ILSpy](http://ilspy.net) - .NET Decompiler.
   * [Reflector](http://www.red-gate.com/products/dotnet-development/reflector/) - .NET Decompiler.
   * [Reflexil](http://sebastien.lebreton.free.fr/reflexil/) - Assembly code editor which can be used as a plugin with other .NET/CLR tools.
+  * [DotNetPELib](https://github.com/LADSoft/DotNetPELib) - Library to read and write .net assemblies in C++11
```

Comment:
Can you please move this line after the Cecil resource link, it kind of messes up the alphabetical order of resources under the section.